src/main/java/io/pakland/mdas/githubstats/infrastructure/shell/components/UserComponent.java

```diff
@@ -0,0 +1,33 @@
+package io.pakland.mdas.githubstats.infrastructure.shell.components;
+
+import io.pakland.mdas.githubstats.infrastructure.shell.validation.DateValidator;
+import io.pakland.mdas.githubstats.infrastructure.shell.validation.UserNameValidator;
+import org.springframework.shell.standard.ShellComponent;
+import org.springframework.shell.standard.ShellOption;
+
+@ShellComponent
+public class UserComponent {
+    
+    private boolean user(
+            @ShellOption(value = {"n"}) String userName,
```

Comment:
I don't like using both single and double dash parameters. I think we can have both, but for the single dash parameters, they should also have a double dash definition (which is what is usually done).

```suggestion
            @ShellOption({"n", "--name"}) String userName,
```

I think it is done like this, after reading the documentaion: https://docs.spring.io/spring-shell/docs/2.1.0-M2/site/reference/htmlsingle/#_customizing_the_named_parameter_keys

Reply:
Done ✅